docs/source/usage.rst

```diff
@@ -74,3 +74,58 @@ Positions:
   - ``H``: OK
   - ``7``: failure to charge HV capacitor
 
+BLE - Bluetooth® Low Energy
+###########################
+
+BLE Heart Rate Service
+----------------------
+
+The MultiGeiger, starting from firmware version 1.15.0-dev, provides a Bluetooth® Low Energy (BLE) service to collect the count rate via a `GATT Heart Rate Service <https://www.bluetooth.com/specifications/gatt/services/>`_ (Service UUID 0x180D). The following characteristics are used:
+
+- 0x2A37 ('Heart Rate Measurement Characteristic'):
+
+  - First byte are status flags, according to the service's standard
+  - 'Heart Rate Measurement' as 16 bit value (little endian), corresponds to Geiger-Müller counts per minute (CPM)
+  - 'Energy Expenditure' as 16 bit value (little endian), represents a rolling packet counter
+- 0x2A38 ('Heart Rate Sensor Position Characteristic')
+
+  - Single byte with TUBE-TYPE value, to convert CPM to radiation rate
```

Comment:
'Tube type' as 8 bit value, ...

Reply:
Aligned wording to other characteristics